tests/ui_tests/test_jobs_page.py

```diff
@@ -0,0 +1,197 @@
+    """Run action on service and validate job in table and popup"""
+    _test_run_action(page, cluster.service_list()[0])
+
+
+def test_provider_action_job(provider: Provider, page: JobListPage):
+    """Run action on host provider and validate job in table and popup"""
+    _test_run_action(page, provider)
+
+
+def test_host_action_job(provider: Provider, page: JobListPage):
+    """Run action on host and validate job in table and popup"""
+    _test_run_action(page, provider.host_create('some-fqdn'))
+
+
+def test_run_successful_job(cluster: Cluster, page: JobListPage):
+    """Run action that finishes successfully and check it is displayed correctly"""
+    expected_info = {
+        'status': JobStatus.SUCCESS,
+        'action_name': SUCCESS_ACTION_DISPLAY_NAME,
+        'object': cluster.name,
+    }
+    with allure.step('Run action and wait it succeeded'):
+        action = cluster.action(display_name=SUCCESS_ACTION_DISPLAY_NAME)
+        run_cluster_action_and_assert_result(cluster, action.name)
+    _check_finished_job_info_in_table(page, expected_info)
+    page.select_filter_success_tab()
+    _check_finished_job_info_in_table(page, expected_info)
+    _check_job_info_in_popup(
+        page, {'status': expected_info['status'], 'action_name': expected_info['action_name']}
+    )
+
+
+def test_run_fail_job(cluster: Cluster, page: JobListPage):
+    """Run action that fails and check it is displayed correctly"""
+    expected_info = {
+        'status': JobStatus.FAILED,
+        'action_name': FAIL_ACTION_DISPLAY_NAME,
+        'object': cluster.name,
+    }
+    with allure.step('Run action and wait it succeeded'):
+        action = cluster.action(display_name=FAIL_ACTION_DISPLAY_NAME)
+        run_cluster_action_and_assert_result(cluster, action.name, status='failed')
+    _check_finished_job_info_in_table(page, expected_info)
+    page.select_filter_failed_tab()
+    _check_finished_job_info_in_table(page, expected_info)
+    _check_job_info_in_popup(
+        page, {'status': expected_info['status'], 'action_name': expected_info['action_name']}
+    )
+
+
+def _test_run_action(page: JobListPage, action_owner: Union[Cluster, Service, Provider, Host]):
+    """
+    Run the "Long" action
+    Check popup info
+    Check table info without filter (All)
+    Activate filter "Running"
+    Check table info
+    """
+    owner_name = action_owner.name if action_owner.__class__ != Host else action_owner.fqdn
+    expected_info = {
+        'status': JobStatus.RUNNING,
+        'action_name': LONG_ACTION_DISPLAY_NAME,
+        'object': owner_name,
+    }
+    with allure.step(
+        f'Run action "{LONG_ACTION_DISPLAY_NAME}" on {action_owner.__class__}'
+    ), page.table.wait_rows_change():
+        long_action = action_owner.action(display_name=LONG_ACTION_DISPLAY_NAME)
+        long_action.run()
+    _check_job_info_in_popup(
+        page, {'status': expected_info['status'], 'action_name': expected_info['action_name']}
```

Comment:
There is also a very important thing with https://arenadata.atlassian.net/browse/ADCM-1802 it will be great to check it here too

Reply:
Do you mean to check if the link to the object that launched the action is correct?

Reply:
For these tests, it will be enough just to check that these links are present and have the correct format.

Reply:
Added, will fail for now